.github/workflows/managed.yml

```diff
@@ -15,6 +15,8 @@ on:
     paths-ignore:
       - "admin/**"
       - "agent/**"
+      - "api-tests/**"
+      - "cli-tests/**"
```

Comment:
What exactly cli-tests are checking?

Reply:
New `pmm` CLI utility. It will replace `pmm-admin`